sdk/python/tests/integration/registration/test_universal_registry.py

```diff
@@ -149,7 +149,7 @@ def pg_registry():
 
     registry_config = RegistryConfig(
         registry_type="sql",
-        path=f"postgresql://{POSTGRES_USER}:{POSTGRES_PASSWORD}@{container_host}:{container_port}/{POSTGRES_DB}",
+        path=f"postgresql+psycopg://{POSTGRES_USER}:{POSTGRES_PASSWORD}@{container_host}:{container_port}/{POSTGRES_DB}",
```

Comment:
Required to instruct that we are using `psycopg3`. If we don't add this, it will try to still use `psycopg2`.

Reply:
Can you elaborate why? In `picnic-feature-store` we do not have this issue and we don't need the `psycopg` part. We shouldn't still have `psycopg2` in the venv anymore, right?

Reply:
Or is it some weird behaviour by SQLAlchemy's `create_engine`, which gets called in the init of `SQLRegistry`? If so, do we need to set something using the `sqlalchemy_config_kwargs` argument of `RegistryConfig`?